app/controllers/items_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
       @merchant = Merchant.find(params[:merchant_id])
       @items = @merchant.items
     else
-      @items = Item.all
+      @items = Item.where(active?: true)
```

Comment:
activerecord filter to ignore inactive items

Reply:
like the use of active record here, we should move this to the Item model

Reply:
Let's move this ActiveRecord login into the model to follow proper MVC.

Reply:
moved to model